build.sbt

```diff
@@ -3,7 +3,7 @@ import BuildHelper._
 inThisBuild(
   List(
     organization := "dev.zio",
-    homepage     := Some(url("https://zio.github.io/zio-prelude/")),
+    homepage     := Some(url("https://zio.dev/zio-prelude/")),
```

Comment:
The page doesn't exist :(
Or will it appear once we merge this PR?

Reply:
There is another PR that will generate the homepage on zio.dev site: https://github.com/zio/zio/pull/7545

Reply:
Good